src/components/Search/SearchAutocomplete.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Autocomplete } from '@material-ui/lab';
+import { createFilterOptions } from '@material-ui/lab/useAutocomplete';
+import TextField from '@material-ui/core/TextField';
+import { getFilterLimit } from './utils';
+import { RegionType, Region } from 'common/regions';
+import MenuItem from './MenuItem';
+
+function getOptionSelected(option: any, selectedOption: any) {
+  return option.fipsCode === selectedOption.fipsCode;
+}
```

Comment:
Both `option` and `selectedOption` should have `Region` type

Reply:
solid. yeah feel free to ignore the `any`s for now